doc/changes/names.inc

```diff
@@ -260,6 +260,10 @@
 
 .. _Abram Hindle: http://softwareprocess.es
 
+.. _Kaisu Lankinen: http://bishoplab.berkeley.edu/Kaisu.html
+
+.. _Padma Sundaram: https://www.nmr.mgh.harvard.edu/user/8071
```

Comment:
@padmasundaram @klankinen feel free to make a PR to master to update these URLs if you want